### DISCUSSION TITLE
security vulnerability: already fixed?

### DISCUSSION THREAD

**Discussion by secretmango** [FossifyOrg/General-Discussion#478] [General]
There was a security vulnerability in simple File manager, is it already fixed?

**Comment by Aga-C** [comment]
Do you mean vulnerability related to PDF Viewer? It's already been fixed around year ago: 
 - https://github.com/SimpleMobileTools/Simple-File-Manager/issues/708#issuecomment-1576159024
- https://github.com/SimpleMobileTools/Simple-File-Manager/issues/619
- https://gitlab.com/fdroid/fdroiddata/-/merge_requests/11496#note_1208626843

**Comment by naveensingh** [comment]
@firefoxlover ping

**Comment by secretmango** [comment]
That long ago? Yes that was it. Thanks!
